src/Microsoft.Azure.WebJobs.ServiceBus/EventHubs/EventHubAttribute.cs

```diff
@@ -27,5 +27,10 @@ public EventHubAttribute(string eventHubName)
         /// </summary>
         [AutoResolve]
         public string EventHubName { get; private set; }
+
+        /// <summary>
+        /// Optional connection name. If missing, tries to use a registered event hub sender.
+        /// </summary>
+        public string Connection { get; set; }
```

Comment:
It's good that this is [AppSetting]. IConnectionProvider just gives it a way to pull the connection from the existing ServiceBusAccountAttribute.  But they seem like they could be used together. 

---
In reply to: [113313470](https://github.com/Azure/azure-webjobs-sdk/pull/1119#discussion_r113313470) [](ancestors = 113313470)